modules/ROOT/pages/migration-from-5x.adoc

```diff
@@ -243,23 +243,27 @@ NOTE: Leaving references to removed plugins in a plugin configuration will cause
 
 See below for specific documentation of potential migration issues, if any, regarding these removed plugins.
 
-Seven `plugins` were removed because their functionality is now a Core part of {productname} 6.0:
+Four plugins were removed as per the previous https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/[{productname} 5 announcement, window="_blank"]:
+
+. `fullpage`
+. `bbcode`
+. `legacyoutput`
+. `tabfocus`
```

Comment:
Maybe copy the note from elsewhere about using tabindex instead?

Reply:
Make a note to do later, I think